packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/dynamoDb-walkthrough.js

```diff
@@ -547,7 +548,7 @@ async function addTrigger(context, resourceName, triggerList) {
 
     if (lambdaResources.length === 0) {
       throw new Error(
-        "No pre-existing functions found in the project. Please use 'amplify add function' command to add a new function to your project."
+        "No pre-existing functions found in the project. Please use 'amplify add function' command to add a new function to your project.",
```

Comment:
"No functions were found in the project. Use 'amplify add function' command to add a new function."